Description: Remove quotes in config file for state_dir value
It creates strange directory /"
Author: Anchal Agarwal <anchalag@amazon.com>
Origin: backport, https://github.com/aws/amazon-ec2-hibinit-agent/commit/d87d6365009f1663247fd8595b487c59d9b121f1
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ec2-hibinit-agent/+bug/2023924
Last-Update: 2023-06-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/hibinit-config.cfg
+++ b/etc/hibinit-config.cfg
@@ -12,7 +12,7 @@
 touch-swap = False
 
 # Location where to create any state files
-state-dir = "/var/lib/hibinit-agent"
+state-dir = /var/lib/hibinit-agent
 
 [swap]
 # If there's no swap then we create it to be equal to the specified
